test/functional/bookmarklet.js

```diff
@@ -25,6 +25,34 @@ describe('BookmarkletController', () => {
 
       let responseData = await response.json()
       responseData.should.have.property('posts')
+      responseData.should.have.property('subscriptions')
+      responseData.subscriptions.should.have.length(1)
     })
+
+    it('should allow create posts in multiple feeds', async () => {
+      let group = await createGroupAsync(luna, 'new-shiny-group')
+
+      let response = await createPostViaBookmarklet(luna, 'Hello, world!', null, null, [luna.username, group.username])
+      response.status.should.eql(200)
+
+      let responseData = await response.json()
+
+      responseData.should.have.property('posts')
+      responseData.should.have.property('subscriptions')
+      responseData.subscriptions.should.have.length(2)
+    })
+
+    it('should ignore nonexistent groups', async () => {
+      let response = await createPostViaBookmarklet(luna, 'Hello, world!', null, null, [luna.username, 'non-existent-group'])
+      response.status.should.eql(200)
+
+      let responseData = await response.json()
+
+      responseData.should.have.property('posts')
+      responseData.should.have.property('subscriptions')
+      responseData.subscriptions.should.have.length(1)
+    })
```

Comment:
Not sure what is best way to handle this kind of error on bookmarklet then. Just show "wrong feeds" or "you can't post into LIST feeds" error message?